pages/team.tsx

```diff
@@ -0,0 +1,30 @@
+import { Heading, Stack, Text } from '@chakra-ui/react';
+import { Trans } from 'react-i18next';
+
+import { useI18n } from '@packages/features/i18n-context';
+
+import Section from '../packages/components/Section';
```

Comment:
Vamos evitar importes relativos (`..`). Temos configuraçoes para facilitar isso:
```suggestion
import Section from '@packages/components/Section';
```

Reply:
O `@` então já faz referência para a raiz?